pkg/term/ascii.go

```diff
@@ -0,0 +1,66 @@
+package term
+
+import (
+	"fmt"
+	"strings"
+)
+
+// ASCII list the possible supported ASCII key sequence
+var ASCII = []string{
```

Comment:
So if we in the future decide to add more ctrl-1~9 detach-keys support, it will be add nine constants here? I mean It's will be a long list and not easy to review and check.

Reply:
LGTM,  looks same as [docker](https://github.com/docker/docker-ce/blob/8a582e89b407639018e067123eb17f61dbfd10ee/components/engine/pkg/term/ascii.go)